### PR TITLE
Handle missing error_doc.problem

### PR DESCRIPTION
**Pull request by kaapstorm**
cf. [FogBugz 167480](http://manage.dimagi.com/default.asp?167480)

Catches situation where error_doc doesn't have an attribute called "problem".

(Also handles situation where no form submissions have been made.)

### CODE REVIEW

**Comment by kaapstorm** [comment]
@snopoke, @dannyroberts, buddy ping @czue.

File: corehq/apps/receiverwrapper/reports.py [hunk 1]
```diff
@@ -115,11 +115,12 @@ def _fmt_date(somedate):
                 time = ServerTime(somedate).user_time(self.timezone).done()
                 return time.strftime(SERVER_DATETIME_FORMAT)
             
+            problem = error_doc.problem if hasattr(error_doc, 'problem') and error_doc.problem else EMPTY_ERROR
```
Comment (snopoke) [inline corehq/apps/receiverwrapper/reports.py]: Could be shortened to `getattr(error_doc, 'problem', None) or EMPTY_ERROR`

Comment (kaapstorm) [inline corehq/apps/receiverwrapper/reports.py]: Nice. Done.

File: corehq/ex-submodules/couchforms/dbaccessors.py [hunk 2]
```diff
@@ -60,25 +60,25 @@ def get_number_of_forms_by_type(domain, type_):
...
-
-    return XFormInstance.view(
+    submissions = XFormInstance.view(
         "couchforms/all_submissions_by_domain",
         startkey=startkey,
         endkey=endkey,
         reduce=True,
-    ).one()['value']
+    ).one()
+    return submissions['value'] if submissions else 0
```
Comment (dannyroberts) [inline corehq/ex-submodules/couchforms/dbaccessors.py]: Oops! Thanks for catching!

**Pull request merged**